config/dynamicconfig/development_pinot.yaml

```diff
@@ -1,15 +1,15 @@
 frontend.enableClientVersionCheck:
   - value: true
 system.advancedVisibilityWritingMode:
-  - value: "triple"
+  - value: "on"
```

Comment:
Ah, where do we configure this? Yaml handles a few boolean operators weirdly (`no` being == false, famously).  Should this be a pure boolean? 

Also, this is a breaking change, what are your thoughts on customers being affected?

Reply:
Our advancedVisibilityWritingMode has multiple values not just true/false, it can also be dual writes (advanced and db).
For the migration, I have added advancedVisibilityMigrationWritingMode which by default will be the dual mode which performs the same as current behavior, so user should not be affected.
This new mode has more values, like only writes to source(ES), only writes to dest (OS), writes to both, and writes to both and db (triple mode).

Reply:
We are using on for advancedVisibilityWritingMode, which means only writes to advanced, during migration it will writes to both advanced visibility stores.
I replaced it with advancedVisibilityMigrationWritingMode in this diff in the visibilityTripleManager to better handle the write mode during migration